tidb-cloud/limited-sql-features.md

```diff
@@ -121,114 +121,110 @@ TiDB Cloud works with almost all workloads that TiDB supports, but there are som
 | `mysql` | `gc_delete_range_done` | Not supported [^4] | Not supported [^4] |
 | `mysql` | `opt_rule_blacklist` | Not supported [^4] | Not supported [^4] |
 | `mysql` | `tidb` | Not supported [^4] | Not supported [^4] |
-| `mysql` | `tidb_ttl_job_history` | Supported | Not supported [^9] |
-| `mysql` | `tidb_ttl_table_status` | Supported | Not supported [^9] |
-| `mysql` | `tidb_ttl_task` | Supported | Not supported [^9] |
 
```

Comment:
mysql-schema.md also needs updates as there is a note stating "The TTL related system tables are not available on [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless) clusters."